src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -325,6 +319,56 @@ await supersededPublications
                 .OnSuccess(publication => _mapper.Map<PublicationViewModel>(publication));
         }
 
+        public async Task<Either<ActionResult, ExternalMethodologyViewModel>> GetExternalMethodology(Guid publicationId)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccessDo(_userService.CheckCanViewPublication)
+                .OnSuccess(publication => publication.ExternalMethodology != null
+                    ? new ExternalMethodologyViewModel(publication.ExternalMethodology)
+                    : NotFound<ExternalMethodologyViewModel>());
+        }
+
+        public async Task<Either<ActionResult, ExternalMethodologyViewModel>> UpdateExternalMethodology(
+            Guid publicationId, ExternalMethodologySaveRequest updatedExternalMethodology)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccessDo(_userService.CheckCanUpdatePublication)
+                .OnSuccess(async publication =>
+                {
+                    _context.Update(publication);
+                    publication.ExternalMethodology ??= new ExternalMethodology();
+                    publication.ExternalMethodology.Title = updatedExternalMethodology.Title;
+                    publication.ExternalMethodology.Url = updatedExternalMethodology.Url;
+                    await _context.SaveChangesAsync();
+
+                    // Clear cache because ExternalMethodology is in Content.Services.ViewModels.PublicationViewModel
+                    await _publicationCacheService.UpdatePublication(publication.Slug);
```

Comment:
Comment could say it updates the cached publication because technically it doesn't clear/delete it anymore. Same on line 365.